kolibri/core/content/utils/resource_import.py

```diff
@@ -232,7 +232,6 @@ def prepare_for_import(self):
         self.resources_before_transfer = (
             ContentNode.objects.filter(channel_id=self.channel_id, available=True)
             .exclude(kind=content_kinds.TOPIC)
-            .values("content_id")
```

Comment:
good cleaning. Happily, Django is smart enough to create the same sql query with or without this line, so same logic with less code :)
I checked it's always this one, with or without `content_id`
`SELECT COUNT(*) AS "__count" FROM "content_contentnode" WHERE ("content_contentnode"."channel_id" = %s AND "content_contentnode"."available" = %s AND NOT ("content_contentnode"."kind" = %s))`

Reply:
Yeah, which means that ironically, what this `values` call was meant to be doing (uniqueifying by `content_id`) was actually doing nothing anyway!